USE-CASES/hazard-annotations.md

```diff
@@ -0,0 +1,135 @@
+
+It could be as coarse as cover any access to the device or specific to the point of selecting a range of operation within the single specific interaction, e.g.:
+- Accessing a connected camera is a privacy hazard
+- Setting an high level of brightness on an halogen lamp might involve risks of overheating it or setting something nearby on fire.
+
+Some devices may provide additional information on hazards they may simply detect and default policies other devices may try to enforce e.g.:
+- A smart meter may notify that the energy flowing through it is over a threshold and the information may trigger a policy that sets some smart switches off.
+
+Some devices may provide information that can be aggregated together to derive/evaluate a risk:
+- A CO₂/VOC sensor may provide a default policy that modulate the smoke and fire hazard evaluation based on its own information, but in itself may not detect smoke or temperature.
+- A connected Oven may not have a smart meter embedded but may provide enough information to desume the energy consumption by the temperature of its heating elements.
+
+### Dependencies - Affected WoT deliverables and/or work items:
+
+The current Thing Description is adequate to support the model, the hazards annotations can be bound to the specific Form Operation for each Interaction Affordance.
+Additional vocabulary terms may be needed to support a finer level of granularity or declare default policies or formulas to derive additional hazard based on the aggregation of information provided by different devices..
+
+### Description:
+
+The user may have a mean of setting policies to restrict interactions with connected devices, e.g.:
+- Stop any interaction with the TV between 3:00PM and 5:00PM if an unattended children is detected.
+- Require stronger authentication to open a door if the user is considered already inside the house.
+
+The home system may automously change the state of a device to reduce the risk level, e.g.:
+- Reduce the brightness of an Halogen Lamp
+- Start the A/C system if a room temperature is outside the bearable range and a person is detected inside.
+
+The same considerations apply to larger scale (e.g. apartment buildings, neighbourhoods, cities) or slightly different contexts such as industry or hospitals.
+
+The risk categories may be diversified to cover field-specific risks, but the structure remains the same:
+- The Thing Description would provide the hazard information
+- Consumer may act on it or enrich/aggregate it and pass it forward for further processing
+
+### Security Considerations:
+
+The system relies on being able to trust the information exchanged and has to detect bad actors.
+
+### Privacy Considerations:
+
+Privacy is one of the possible category of hazards that may be accounted for, it is no different from a physical hazard in this use-case.
```

Comment:
Not completely true.  Privacy and Invasion are related in that the hazard is *other people* and privacy in particular is related to leaking of information to other people.   So the different thing about privacy is that it matters where a given information flow goes.  A camera is a hazard because the information flow MAY be leaked to someone outside the set of people the owner wants to provide it to.  But it's not a problem if that information goes to the intended recipient.  Something like a fire hazard only involves the device and the user (although even then a fire hazard only means a fire MAY happen in certain circumstances).

One thing missing by the way is a definition of "hazard".   A hazard is a *risk* of something bad happening.  You might want to dig up the ISO definition and put it somewhere.

Reply:
https://advisera.com/45001academy/blog/2016/03/23/hazards-vs-risks-what-is-the-difference-according-to-disiso-45001/#:~:text=What%20is%20a%20Hazard%20in,the%20ability%20to%20harm%20individuals%3F

Reply:
ISO/DIS 45001:
hazard: “source or situation with a potential to cause injury and ill health” 
- This standard is for "operational hazards" e.g. in a business, for workplace safety, but still useful.
- You might have to expand it a bit to include "violation of privacy" under "injury".
- This document also has a useful list of hazard categories.

Reply:
> Not completely true. Privacy and Invasion are related in that the hazard is _other people_ and privacy in particular is related to leaking of information to other people. So the different thing about privacy is that it matters where a given information flow goes. A camera is a hazard because the information flow MAY be leaked to someone outside the set of people the owner wants to provide it to. But it's not a problem if that information goes to the intended recipient. Something like a fire hazard only involves the device and the user (although even then a fire hazard only means a fire MAY happen in certain circumstances).

The hazards are processed exactly in the same way no matter the kind. e.g. the oven may roast a chicken or roast a wooden block you left inside, potentially causing a fire. It is not different from a connected camera streaming to the wrong endpoint at the wrong time.

> 
> One thing missing by the way is a definition of "hazard". A hazard is a _risk_ of something bad happening. You might want to dig up the ISO definition and put it somewhere.

Sounds a good idea, thank you :)